### PR TITLE
Scalable Revocation Method(s) for Verifiable Credentials

### PR DESCRIPTION
**Pull request by coffeethecup**

### CODE REVIEW

File: work_items/revocation_methods_for_verifiable_credentials_.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Work Item Owners
+- Andreas Freitag (andreas.freitag.77@gmail.com)
+- Mike Lodder (redmike7@gmail.com)
+
+## External Partners (OPTIONAL)
+**This is only for work items intended for transition to an external standards
+body after incubation here. This may be filled in later and is not required for
+initial approval of the work item.**:
+- W3C
```
Comment (dhuseby) [inline work_items/revocation_methods_for_verifiable_credentials_.md:25]:
```suggestion
- IETF
```

File: work_items/revocation_methods_for_verifiable_credentials_.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Revocation Methods for Verifiable Credentials
+
+The goal is to identify and define revocation methods for verifiable credentials and create the protocol to ensure interoperability.
```
[the remark this answers is not in the document]
Comment (dhuseby) [reply work_items/revocation_methods_for_verifiable_credentials_.md:3]: Revocation methods are independent applications of cryptography. The term verifiable credentials implies W3C and linked data and all of that. Revocation is about cryptographic proofs of what is in the set of non-revoked data and what is not. There is no implied commitment to any existing data encoding standard other than cryptographic material encoding such as the work item: [https://github.com/decentralized-identity/crypto-wg/blob/main/work_items/cbb_data_encoding.md](https://github.com/decentralized-identity/crypto-wg/blob/main/work_items/cbb_data_encoding.md)

File: work_items/revocation_methods_for_verifiable_credentials_.md (hunk 1 above)
Comment (dhuseby) [reply work_items/revocation_methods_for_verifiable_credentials_.md:25]: I think this work is more suitable for Internet-level engineering. It is no more a web technology than digital signatures are.

**Review by dhuseby — CHANGES_REQUESTED** [review]

File: work_items/revocation_methods_for_verifiable_credentials_.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+In this work item, possible revocation methods should be collected and assessed. Suitable revocation methods should be defined in a revocation protocol. 
+
+Tasks:
+- Collect possible revocation methods
+  - Cryptographic accumulator suggestion by Mike (https://hackmd.io/O4c3wiLZQLeXuXirm7dl9A#Math-explaination)
+  - tbd
+
+- Define an assessment setup (#of credentials, #of revocations per epoch,...)for performance testing
```
[the remark this answers is not in the document]
Comment (brentzundel) [reply work_items/revocation_methods_for_verifiable_credentials_.md:11]: perhaps: 
```suggestion
- Define an assessment setup (#of entries, #of revocations per epoch,...)for performance testing
```

File: work_items/revocation_methods_for_verifiable_credentials_.md (hunk 1 above)
Comment (coffeethecup) [reply work_items/revocation_methods_for_verifiable_credentials_.md:26]: Not sure if we have no overlap to W3C. I would suggest to keep it in the list.

**Review by brentzundel — APPROVED** [review]

File: work_items/revocation_methods_for_verifiable_credentials_.md (hunk 1 above)
Comment (dhuseby) [reply work_items/revocation_methods_for_verifiable_credentials_.md:26]: @coffeethecup I'm ok with making the list both W3C and IETF

**Pull request merged**